src/ConvertToSarif/Program.cs

```diff
@@ -25,18 +26,23 @@ public static int RunConvertFile(ConvertOptions convertOptions)
         {
             try
             {
-                ToolFormatConversionOptions toolFormatConversionOptions = 0;
+                LoggingOptions loggingOptions = LoggingOptions.None;
 
                 if (convertOptions.PrettyPrint)
                 {
-                    toolFormatConversionOptions |= ToolFormatConversionOptions.PrettyPrint;
+                    loggingOptions |= LoggingOptions.PrettyPrint;
                 };
 
                 if (convertOptions.Force)
                 {
-                    toolFormatConversionOptions |= ToolFormatConversionOptions.OverwriteExistingOutputFile;
+                    loggingOptions |= LoggingOptions.OverwriteExistingOutputFile;
```

Comment:
>OverwriteExistingOutputFile [](start = 53, length = 27)

Why not "Force"? That's pretty conventional for meaning "overwrite existing file".

Reply:
'Force' could be a star wars reference, that's why. 'OverwriteExistingOutputFile' is perfectly descriptive. Think of it as an XML doc comment that's burned RIGHT INTO THE NAME, hee,

---
In reply to: [136626110](https://github.com/Microsoft/sarif-sdk/pull/614#discussion_r136626110) [](ancestors = 136626110)